### PR TITLE
Issue #2917 Fixing the CVE-2018-1000201 for docs/Gemfile.lock

### LINKED ISSUE #2917: Update the ffi/ffi version in docs/Gemfile.lock
Github UI reported that the version of `ffi` we are using  in `docs/Gemfile.lock` as a dependency has a security issue i.e. CVE https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1000201

In `docs/Gemfile.lock` we have `ffi/ffi (>= 1.3.0)`

screen shot 2018-10-22 at 3 51 22 pm

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Updating the Gemfile.lock to latest versions.

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

Fixes #2917

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
Updated the Gemfile by `$ bundle update` command.

**Comment by anjannath** [comment]
retest this please

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
Want to make sure doc build is successful by CI

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -21,27 +21,27 @@ GEM
...
     ethon (0.11.0)
       ffi (>= 1.3.0)
-    eventmachine (1.2.5)
+    eventmachine (1.2.7)
     execjs (2.7.0)
     fast_blank (1.0.0)
-    fastimage (2.1.1)
-    ffi (1.9.23)
+    fastimage (2.1.4)
+    ffi (1.9.25)
```
Comment (praveenkumar) [inline docs/Gemfile.lock]: 👍

**Pull request merged**